packages/module/src/components/edges/DefaultEdge.tsx

```diff
@@ -112,6 +113,8 @@ const DefaultEdgeInner: React.FunctionComponent<DefaultEdgeInnerProps> = observe
     return null;
   }
 
+  const detailsLevel = useDetailsLevel();
```

Comment:
```suggestion
  const detailsLevel = element.getGraph().getDetailsLevel();
```

Reply:
This is the same issue that nodes is having. It doesn't cause an issue here due to the edge also updating on a scale change but if that were to change than this would be an issue, better to fix it now.